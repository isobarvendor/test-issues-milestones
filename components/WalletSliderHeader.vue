<template>
  <div class="slider-header">
    <div class="title">
      <slot></slot>
    </div>
  </div>
</template>

<script>
export default {
  name: "WalletSliderHeader",
};
</script>
<style lang="scss">
.wallet{
  @media only screen and (max-width: 767px) {
    .slider-header{
      .title{
        font-size: 20px;
      }
    }
  }
}
</style>
