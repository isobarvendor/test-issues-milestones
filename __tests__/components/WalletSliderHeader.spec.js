
import { shallowMount } from '@vue/test-utils'

import WalletSliderHeader from "@/components/WalletSliderHeader";


const factory = () => {
  return shallowMount(WalletSliderHeader);
};

describe("WalletSliderHeader", () => {


  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.vm).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
