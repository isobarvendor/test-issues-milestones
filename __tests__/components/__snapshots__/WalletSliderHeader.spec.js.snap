// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WalletSliderHeader renders properly 1`] = `
<div class="slider-header">
  <div class="title"></div>
</div>
`;
